tests/ops/op_math/test_sum.py

```diff
@@ -660,6 +660,9 @@ def test_hash(self):
         op3 = Sum(qml.PauliX("a"), qml.PauliY("b"), qml.PauliZ(-1))
         assert op3.hash != op1.hash
 
+        op3 = Sum(qml.X("a"), qml.X("a"), qml.Y("b"))
+        assert op3.hash != op1.hash
```

Comment:
```suggestion
        op4 = Sum(qml.X("a"), qml.X("a"), qml.Y("b"))
        assert op4.hash != op1.hash
```
Really nit-picky thing, but I think would it make the code clearer to use a different variable for this operator.